hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/s3/multipart/S3MultipartUploadCompleteRequest.java

```diff
@@ -630,4 +634,15 @@ public static OMRequest blockMPUCompleteWithBucketLayoutFromOldClient(
     }
     return req;
   }
+
+  private String multipartUploadedKeyHash(
+      OmMultipartKeyInfo.PartKeyInfoMap partsList) {
+    StringBuffer keysConcatenated = new StringBuffer();
+    for (PartKeyInfo partKeyInfo: partsList) {
+      keysConcatenated.append(partKeyInfo.getPartName());
+    }
+    return DigestUtils.md5Hex(keysConcatenated.toString()) + "-"
```

Comment:
How does this differentiate betweek 2 multipart uploads with varying data but the same metadata?